tlovett1/safe-redirect-manager
# Custom protocol per route

### Description of the Change
Custom protocol selection feature per route implemented. 


Closes #46 

### How to test the Change
- Add/edit a redirect rule
- Choose `http` or `https` in Redirect Protocol section
- Visit the route
- Check if it redirects to the protocol you've selected. 

### Changelog Entry

> Added - Custom redirect protocol per route


### Credits

Props @jayedul 


### Checklist:
